RELEASE_NOTES.md

```diff
@@ -1,6 +1,51 @@
-_This file holds "in progress" release notes for the current release under development and is intended for consumption by the Chef Documentation team. Please see <https://docs.chef.io/release_notes.html> for the official Chef release notes._
+This file holds "in progress" release notes for the current release under development and is intended for consumption by the Chef Documentation team. Please see <https://docs.chef.io/release_notes.html> for the official Chef release notes.
 
-# Chef Client Release Notes 14.0.202
+# Chef Client Release Notes 14.1:
+
+## Windows Task
+
+The `windows_task` resource has been entirely rewritten. This resolves a
```

Comment:
we can also mention about resolving idempotency issues.

Reply:
Thanks, have done so.